clientd3d/music.c

```diff
@@ -647,7 +782,16 @@ void MusicAbort(void)
 void MusicStart(void)
 {
    if (state == STATE_GAME)
-      PlayMusicRsc(bg_music);
+   {
+      if (latest_music != bg_music){
```

Comment:
Change this to "latest_music == bg_music" and flip the then and else clauses.

Reply:
Thanks. Done.